### DISCUSSION TITLE
Add permit available to public search terms

### DISCUSSION THREAD

**Discussion by campmlc** [ArctosDB/arctos#4967] [General]
Currently this is only searchable with operator access. We are funneling this info to GGBN using the vocabulary "permit available, permit not required"; also "permit type". It would be good to have the same search parameters available in Arctos, at least for tissues collected after 2014, for Nagoya purposes.

**Comment by dustymc** [comment]
This is a currently social problem; I'm un-assigning myself until it becomes a technical problem.

- Do we want to provide permit search functionality for non-operator users?
- If so, by what criteria?
- Do we want to display something about permits somewhere in results or specimen detail? 
- If so, what and where?

**Comment by jldunnum** [comment]
I think I would prefer to keep most details within the management access level.  Don't see why public should be searching our permits.


______________________________________________________________

Jonathan L. Dunnum Ph.D.
Senior Collection Manager
Division of Mammals, Museum of Southwestern Biology
University of New Mexico
Albuquerque, NM 87131
(505) 277-9262
Fax (505) 277-1351

MSB Mammals website: http://www.msb.unm.edu/mammals/index.html
Facebook: http://www.facebook.com/MSBDivisionofMammals

Shipping Address:
Museum of Southwestern Biology
Division of Mammals
University of New Mexico
CERIA Bldg 83, Room 204
Albuquerque, NM 87131
________________________________

**Comment by campmlc** [comment]
To clarify, I am suggesting we only add the fields that we are already
making publicly available via GGBN. This would allow researchers to use
both Arctos and GGBN to find the same info. These fields are "permit
available, permit not required, and permit type". This information is
required to comply with the Nagoya protocol.

**Comment by dustymc** [comment]
Tabling, doesn't seem to be interest.

**Comment by campmlc** [comment]
Still relevant. @jldunnum
